src/main/java/se/sundsvall/seabloader/service/InvoiceService.java

```diff
@@ -34,11 +34,12 @@ public InvoiceService(InvoiceRepository invoiceRepository, InvoicePdfMerger invo
 		this.invoiceCacheClient = invoiceCacheClient;
 	}
 
-	public void create(final byte[] content) {
+	public void create(final String municipalityId, final byte[] content) {
 		final var invoiceEntity = toInvoiceEntity(content);
```

Comment:
`toInvoiceEntity` could take the municipalityId as an input parameter, in order to enforce the setting of the value.
(it will minimize the risk of forgetting this value).